packages/rest/src/__tests__/acceptance/routing/routing.acceptance.ts

```diff
@@ -735,6 +735,61 @@ describe('Routing', () => {
         .get('/greet/world')
         .expect(200, 'HELLO WORLD');
     });
+
+    it('redirect swagger routes', async () => {
+      const app = new RestApplication();
+      await whenIMakeRequestTo(app)
+        .get('/swagger-ui')
+        .expect(308);
+      await whenIMakeRequestTo(app)
+        .get('/explorer')
+        .expect(308);
```

Comment:
Please verify the redirect target too (i.e. the value returned in `Location` header).

Reply:
test removed